tests/e2e/util/image/image.go

```diff
@@ -28,18 +28,17 @@ import (
 )
 
 var (
-	TiDBPreviousVersions []string = []string{"v5.0.6", "v5.1.4", "v5.2.4", "v5.3.2", "v5.4.2"}
+	TiDBPreviousVersions []string = []string{"v6.1.7", "v6.5.10", "v7.1.5", "v7.5.2", "v8.1.0"}
 )
 
 const (
 	// TiDB Version
-	TiDBLatestPrev     = "v6.0.0"
-	TiDBLatest         = "v6.1.0"
+	TiDBLatestPrev     = "v7.5.2"
+	TiDBLatest         = "v8.1.0"
```

Comment:
bump DM for 

Error: [Lightning:Restore:ErrCreateSchema]create schema failed, table: `dm_cluster_5265`.``, stmt: restore database schema: run create schema job failed: Error 1273: Unsupported collation when new collation is enabled: 'utf8mb4_0900_ai_ci'